### DISCUSSION TITLE
无法授权设备使用专业版

### DISCUSSION THREAD

**Discussion by Jameshening** [getlantern/lantern-forum-cn#831] [Q&A]
苹果手机用美区账号下载了蓝灯，在安卓手机客户端购买了专业版；目前通过电子邮件来授权iOS客户端使用专业版，输入电子邮箱收到的验证码后，显示“验证码无效或过期”！重复这一步骤，仍然收到同样验证码，无法实现授权，请帮助解决，谢谢

**Comment by Jameshening** [comment]
![37772208c4f6b9d7bffe0b8542db1421](https://github.com/user-attachments/assets/8e157706-cdd7-4789-a455-151c6663ba92)

**Comment by Jameshening** [comment]
![Uploading IMG_7038.png…]()

**Comment by Jameshening** [comment]
![Uploading IMG_7038.jpeg…]()

**Comment by MaxMKMax** [comment]
@Jameshening 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by MaxMKMax** [comment]
@Jameshening 请您也提供我支付单据和账号邮箱. 谢谢. 我帮您提交问题上去看怎么解决.
